docs/index.md

```diff
@@ -1,19 +1,20 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: PlanPal
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+* **PlanPal** is an address book designed for student event planners at NUS who need to manage contacts (e.g., attendees, vendors, sponsors, and volunteers) for their events. 
+* PlanPal offers a centralized platform to organize, track, and access contact information, ensuring efficient coordination and smooth communication throughout the event planning process.
+* While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
 * If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
 * If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
 
-
 **Acknowledgements**
 
```

Comment:
Would it be good if you could add this acknowledgement according to what the CS2103T website requested too?

> A3.
> Acknowledge the original source of the code e.g.,
> This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).

Reply:
Thanks Jun Yu for your suggestion, I agree that it is important and will implement the changes!